.github/workflows/test.yaml

```diff
@@ -32,24 +32,17 @@ jobs:
         run: |
           git config --global user.name "scverse-bot"
           git config --global user.email "108668866+scverse-bot@users.noreply.github.com"
-      - name: Set up Python ${{ matrix.python }}
-        uses: actions/setup-python@v5
+      - name: Install uv
+        uses: astral-sh/setup-uv@v5
         with:
-          python-version: ${{ matrix.python }}
-          cache: "pip"
-          cache-dependency-path: "{{cookiecutter.project_name}}/pyproject.toml"
+          cache-dependency-glob: "{{cookiecutter.project_name}}/pyproject.toml"
       - name: Install Ubuntu system dependencies
         if: matrix.os == 'ubuntu-latest'
         run: sudo apt-get install pandoc
-      - name: Install build & check dependencies
-        run: python -m pip install --upgrade pip wheel cookiecutter pre-commit hatch
       # Build
       - name: Build from template
-        shell: python
         run: |
-          from cookiecutter.main import cookiecutter
-          overrides = dict(package_name='${{ matrix.package-name }}')
-          cookiecutter('.', no_input=True, extra_context=overrides)
+          uvx --with pre-commit cruft create  . --no-input --extra-context '{"package_name": "${{ matrix.package-name }}"}'
       # Check
       - name: Set up pre-commit cache
         uses: actions/cache@v4
```

Comment:
Does this still do anything? I'd guess yes, because I don't think pre-commit uses `uv` to create environments?

Reply:
unless we also install `pre-commit-uv` which we could: https://github.com/tox-dev/pre-commit-uv

Reply:
Let's do that. It's maintained by tox and if it breaks the cost is low because it's just in the main repo, not the template.